Cargo.toml

```diff
@@ -79,6 +79,9 @@ no_neon = []
 
 zeroize = ["zeroize_crate", "arrayvec/zeroize"]
 
+# Enable optional serialization and deserialization of the Hash type using serde
+serde = ["dep:serde"]
```

Comment:
This is equivalent to the default behavior if we just omit this line, right? If we want to keep it, we should move it above the comment that says "Features below this line are undocumented and unstable".

Reply:
That's a nice trick, didn't know about it. Made the change as requested (line removed)